vagrant/provisioning/roles/foia/templates/arkcase-portal-server.yaml

```diff
@@ -0,0 +1,40 @@
+logging.level.web: DEBUG
+
+# server.port=8080
+# server.servlet.context-path=/${portal.applicationName}
+
+# Property used for enable/disable Spring security. For authenticated portal comment this property.  
+#spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration,org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration
+
+
+portal.externalConfiguration: true
+portal.host={{ external_host }}
```

Comment:
This file extension is ".yaml" but it looks like a .properties file.

YAML: I would expect to see 'portal.host: {{ external_host }}' but instead I see an equals sign.

Is it OK?  

Actually I see some lines in YAML (colon space) and other lines in properties format (equals sign).

Reply:
I'll fix these.